src/transform.ts

```diff
@@ -135,8 +133,7 @@ export interface WindowTransform {
   window: WindowFieldDef[];
 
   /**
-   * The frame for the window, if none is set the default is `[null, 0]` everything before the
-   * current item.
+   * A two element specification about how large the sliding window is. The first element indicates the start and the second element indicates the end. If `null` is specified for the start, it will include everything before the current point. If `null` is specified for the end, it will include everything after the endpoint. For example a frame of `[-5,5]` says the window should include 5 previous objects and 5 after objects. The default is `[null, 0]`, which means include everything in the window. `[null, null]` would mean include everything in the window.
```

Comment:
> The default is `[null, 0]`, which means include everything in the window. `[null, null]` would mean include everything in the window. 

This is inaccurate.  Your text basically says that `[null, 0]` and `[null,null]` are the same.   Why not just copy text from  Vega docs https://vega.github.io/vega/docs/transforms/window/.

- [ ] Maybe use "data objects" over just "objects" though
- [ ] Another thing to change is that we always use the following format for default values:

```

__Default value:__:  `[null, 0]` (includes the current object and all preceding objects)
```

(Please make sure that you use this format for all properties.)

Reply:
Sorry, I thought the default was `[null, null]` realized it was `[null, 0]` so I quickly changed it, but forgot to update the description. I'll update this.